### PR TITLE
Dem diagonal wall impact example

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- We did not have any simple validation of the contact model for contact  with angles

# Description of the solution

- Add one derived from the MFIX05 test benchmark

# How Has This Been Tested?

- Results are very good when compared to experiments and to the MFIX benchmark

# Documentation

- Fully documented the example

### CODE REVIEW

File: doc/source/examples/dem/oblique-wall-impact/images/coeff_restitution.png
Comment (OGaboriault) [inline doc/source/examples/dem/oblique-wall-impact/images/coeff_restitution.png:1]: This is the only graph that has MFIX in it. Shouldn't we remove MFIX here or add MFIX in the other two graphs?

**Review by OGaboriault — APPROVED** [review]
Good work. This example is answering a lot of questions about the tangential damping like we already talked about. I just have one small comment. :)

File: doc/source/examples/dem/oblique-wall-impact/images/coeff_restitution.png
Comment (blaisb) [reply doc/source/examples/dem/oblique-wall-impact/images/coeff_restitution.png:1]: Well it's the only one where the agreement is not as good. Hence why I wanted to put the MFIX results just to show that everybody is bad here. For the other graph, the MFIX results and ours are in perfect agreement with the experiments

File: doc/source/examples/dem/oblique-wall-impact/oblique-wall-impact.rst [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+Generates 34 file with the prefix ``run_oblique_collision`` and the ``{angle}.prm`` as a suffix. 
+
+----------------------
+Running the Simulation
+----------------------
+Once all files are created, the simulation can be launched in parallel using the following command:
+
+.. code-block:: text
+
+  for i in $(ls run_oblique_collision*) ; do dem $i&; sleep2; done
```
Comment (OGaboriault) [inline doc/source/examples/dem/oblique-wall-impact/oblique-wall-impact.rst:132]:
```suggestion
  for i in $(ls run_oblique_impact*); do dem $i & sleep 2; done
```

**Review by OGaboriault — APPROVED** [review]

**Pull request merged**